### DISCUSSION TITLE
Docker Compose Image is still 1.1?

### DISCUSSION THREAD

**Discussion by xanthos84** [anonaddy/docker#281] [General]
hi,
docker pull does not get version 1.2 for anonaddy...
when does the image gets an update to newest version?
thank you

**Comment by Dominion0815** [comment]
@crazy-max Meanwhile there is v1.2.1, when will there be a new Docker image?

**Comment by xanthos84** [comment]
also waiting for the docker image to be updated....

**Comment by Dominion0815** [comment]
v1.2.2 ping @crazy-max  ;)
